test/integration/apiserver/admissionwebhook/admission_test.go

```diff
@@ -90,28 +90,29 @@ var (
 
 	// customTestFuncs holds custom test functions by resource and verb.
 	customTestFuncs = map[schema.GroupVersionResource]map[string]testFunc{
-		gvr("", "v1", "namespaces"):                          {"delete": testNamespaceDelete},
-		gvr("apps", "v1beta1", "deployments/rollback"):       {"create": testDeploymentRollback},
-		gvr("extensions", "v1beta1", "deployments/rollback"): {"create": testDeploymentRollback},
+		gvr("", "v1", "namespaces"):                                         {"delete": testNamespaceDelete},
+		gvr("apps", "v1beta1", "deployments/rollback"):                      {"create": testDeploymentRollback},
+		gvr("extensions", "v1beta1", "deployments/rollback"):                {"create": testDeploymentRollback},
+		gvr("authentication.k8s.io", "v1", "tokenreviews"):                  {"create": testNonPersistentResourceCreate},
```

Comment:
nit, add a blank line here to group all the types that use `testNonPersistentResourceCreate` (will need to be rebased anyway)